src/core/logger/consolelogger.cpp

```diff
@@ -0,0 +1,88 @@
+/***********************************************************************************************************************
+**
+** Copyright (C) 2022 BaseALT Ltd. <org@basealt.ru>
+**
+** This program is free software; you can redistribute it and/or
+** modify it under the terms of the GNU General Public License
+** as published by the Free Software Foundation; either version 2
+** of the License, or (at your option) any later version.
+**
+** This program is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with this program; if not, write to the Free Software
+** Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+**
+***********************************************************************************************************************/
+
+#include "consolelogger.h"
+
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <unistd.h>
+
+namespace gpui
+{
+namespace logger
+{
+void ConsoleLogger::logDebug(const LoggerMessage &message)
+{
+    std::string prefix = "DEBUG";
```

Comment:
Prefixes could be constant.

Reply:
We could have list of prefixes for all logger classes defined in one header, of in a base class.